README.md

```diff
@@ -1,26 +1,28 @@
-CodeMirror-mode-promql
-======================
+CodeMirror-promql
+=================
 [![CircleCI](https://circleci.com/gh/prometheus-community/codemirror-promql.svg?style=shield)](https://circleci.com/gh/prometheus-community/codemirror-promql) [![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](./LICENSE)
 [![NPM version](https://img.shields.io/npm/v/codemirror-promql.svg)](https://www.npmjs.org/package/codemirror-promql)
 
 ## Overview
-This project provides a mode for [CodeMirror Next](https://codemirror.net/6) that handles syntax highlighting and autocompletion for the PromQL ([Prometheus Query Language](https://prometheus.io/docs/introduction/overview/)).
+This project provides a mode for [CodeMirror Next](https://codemirror.net/6) that handles syntax highlighting, linter and autocompletion for the PromQL ([Prometheus Query Language](https://prometheus.io/docs/introduction/overview/)).
 
-Initially this code was in a private repository. It has been transferred to the `prometheus-community` organization (Thanks to [Julius Volz](https://github.com/juliusv) who helped us with that).
+> Initially this code was in a private repository. It has been transferred to the `prometheus-community` organization (Thanks to [Julius Volz](https://github.com/juliusv) who helped us with that).
 During the transfer, the repository and the package changed its name from `codemirror-mode-promql` to the current one: `codemirror-promql`.
 
 ### Status
 This mode is **not** production ready. If you want to use it, you may encounter some issues.
+API and usage of this mode can be deeply changed between two versions.
 
 See the [Development](#development) section for more information on features and usage.
 
 ### Installation
-This mode is available as an npm package:
+This mode is available as a npm package:
 
-| Version             | command to use                        |
-| ------------------- | ------------------------------------- |
-| >= 0.3.0            | `npm install codemirror-promql`       |
-| >= 0.2.2 < 0.3.0    | `npm install codemirror-mode-promql`  |
+```bash
+npm install codemirror-promql
+```
+
+**Note:** You will have to install on your own side [CodeMirror Next](https://codemirror.net/6)
```

Comment:
```suggestion
**Note:** You will have to manually install [CodeMirror Next](https://codemirror.net/6) as a peer dependency.
```

Reply:
@juliusv I think the "as a peer dependency" may not be exactly what you mean. CodeMirror Next is the peer dependency of this library, so the user must install CMN themselves, but as a normal dependency, not as a peer dependency (of the user's app)?

Reply:
Oh yeah, good point. So maybe just:

```suggestion
**Note:** You will have to manually install [CodeMirror Next](https://codemirror.net/6), as it is a peer dependency to this package.
```